test/e2e/v1/e2e_suite.go

```diff
@@ -108,7 +108,8 @@ var _ = Describe("kubebuilder", func() {
 
 			// NOTE: If you want to run the test against a GKE cluster, you will need to grant yourself permission.
 			// Otherwise, you may see "... is forbidden: attempt to grant extra privileges"
-			// $ kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=cluster-admin --user=myname@mycompany.com
+			// $ kubectl create clusterrolebinding myname-cluster-admin-binding \
+			// --clusterrole=cluster-admin --user=myname@mycompany.com
```

Comment:
```suggestion
			//     --clusterrole=cluster-admin --user=myname@mycompany.com
```

Visual indent

Reply:
1 of 100 comments requests to add \ and remove the spaces.
IHMO I prefer this other suggestion if you do not mind.

Reply:
Did I request to remove the spaces? Well it seems I changed my mind sorry for that. The `/` is needed but thats already there.